src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -24,7 +26,41 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
     You may both have to come to a court hearing, if they apply.`,
     line3: 'If you wait a year before applying then you will need to explain the delay to the court.',
   },
-  checkboxLine: `I want to ${isDivorce ? 'finalise my divorce' : 'end my civil partnership'}`,
+  readMoreJoint: {
+    subHeader: 'Changing to a sole application',
+    subHeader2: 'If you want to change to a sole application now',
+    line1: `If you still want to ${
+      isDivorce ? 'get divorced' : 'end your civil partnership'
+    } but you do not think your ${partner} will confirm the final order then you can change to a sole application.
+    This means your ${partner} will become the respondent.`,
+    line2: `You should still continue and confirm you want to ${
+      isDivorce ? 'finalise your divorce' : 'end your civil partnership'
+    } below.
+    If your ${partner} does not also confirm within 2 weeks then you will be able to ${
+      isDivorce ? 'finalise your divorce' : 'end your civil partnership'
+    } as a sole applicant.
+    You will receive an email with information on how to change your application, if they do not apply.`,
+  },
+  readMoreChangeToSole: {
+    line1: `If you know your ${partner} is not going to confirm the joint application, then you can apply to change to a sole application now.
+    You should save and sign out and follow these steps.`,
+    orderedList1FinalOrder: `<a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/952032/d36-eng.pdf"
```

Comment:
remove the tags here and have it in the html like what we do for other links.

Reply:
completed.